ivy/array/general.py

```diff
@@ -173,6 +173,44 @@ def gather_nd(
         """
         return ivy.gather_nd(self, indices, out=out)
 
+    def scatter_nd(self: ivy.Array, 
+        indices:Union[ivy.Array, ivy.NativeArray], 
+        updates:Union[ivy.Array, ivy.NativeArray],
+        shape: Union[ivy.Array, ivy.NativeArray] = None,
+        reduction: str = "sum",
+        *,
+        out: Optional[ivy.Array] = None,
+        ) -> ivy.Array:
+        """
```

Comment:
Please comply with styling, e.g. `self: ivy.Array,` should be on a new line, and the output should have the same indent as `def`.